flake.nix

```diff
@@ -157,7 +157,7 @@
           text = ''
             shopt -s globstar
             clang-format --dry-run --Werror ${srcGlob}
-            clang-tidy ${srcGlob}
+            #clang-tidy ${srcGlob}
```

Comment:
Maybe good to add a TODO comment in here to re-enable once clang tidy is fixed upstream?